moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/status_codes.h

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ *      Title     : status_codes.h
+ *      Project   : moveit_jog_arm
+ *      Created   : 2/25/2019
+ *      Author    : Andy Zelenak
+ *
+ * BSD 3-Clause License
+ *
+ * Copyright (c) 2019, Los Alamos National Security, LLC
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * * Redistributions of source code must retain the above copyright notice, this
+ *   list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ *
+ * * Neither the name of the copyright holder nor the names of its
+ *   contributors may be used to endorse or promote products derived from
+ *   this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************/
+
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+namespace moveit_jog_arm
+{
+enum StatusCode : int8_t
+{
+  kNoWarning = 0,
+  kSingularityDecelerate = 1,
+  kSingularityHalt = 2,
+  kCollision = 3,
+  kJointBound = 4
+};
+
+const std::unordered_map<uint, std::string>
+    kStatusCodeMap({ { kNoWarning, "No warnings" },
```

Comment:
MoveIt Style would dictate that this should be named STATUS_CODE_MAP. 'Status code' is a little generic but since this is in the `moveit_jog_arm` namespace, I think it works. Nevertheless, I might consider naming it something like, `JOG_ARM_STATUS_CODE_MAP`

Reply:
ok, JOG_ARM_STATUS_CODE_MAP it is